frontend/public/kubevirt/components/dashboards/utils.js

```diff
@@ -16,7 +16,7 @@ export const getPrometheusQuery = async(query) => {
   return coFetchJSON(url);
 };
 
-export const fetchPeriodically = async(url, onFetch, responseHandler, fetchMethod = coFetchJSON) => {
+export const fetchPeriodically = async(url, onFetch, onTimeoutScheduled, responseHandler, fetchMethod = coFetchJSON, ) => {
```

Comment:
can you please remove the comma ? 

```suggestion
export const fetchPeriodically = async(url, onFetch, onTimeoutScheduled, responseHandler, fetchMethod = coFetchJSON) => {
```